server/src/pages/homepage/view.js

```diff
@@ -558,6 +558,23 @@ class Body extends React.Component {
     window.location = "/shots";
   }
 
+  renderGetFirefox() {
+    if (this.props.userAgent.search(/firefox\/\d{1,255}/i) === -1) {
```

Comment:
The logic would change based on the above comment, but generally I try to put exceptions first, so like:

```js
renderGetFirefox() {
  if (!this.props.isFirefox) {
    return null;
  }
  return ...
}
```